_CodingChallenges/051.1-astar.md

```diff
@@ -32,6 +32,14 @@ contributions:
     source: "https://github.com/MkLahane/Game-Development"
     author:
       name: "Makarand Lahane"
+  - title: "Pathfinding A* in Windows Form using C#"
+    author:
+      name: "Trong Hieu Nguyen"
+      url:
+    url: "https://github.com/TrongHieu90/PathfindingAStar"
+    source: "https://github.com/TrongHieu90/PathfindingAStar"
+    
```

Comment:
Line `41` and `42` contain whitespace. Tests should pass as soon as they are removed!

Reply:
White Space has been removed in these 2 lines.